src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2727,60 +2722,51 @@ std::string UsgsAstroLsSensorModel::constructStateFromIsd(const std::string imag
    state["m_mountingMatrix"][7] = sin_a * cos_b;
    state["m_mountingMatrix"][8] = cos_a * cos_b;
 
-   state["m_dtEphem"] = isd.at("DT_EPHEM");
-   state["m_t0Ephem"] = isd.at("T0_EPHEM");
-   state["m_dtQuat"] =  isd.at("DT_QUAT");
-   state["m_t0Quat"] =  isd.at("T0_QUAT");
-
-   state["m_numEphem"] = isd.at("NUMBER_OF_EPHEM");
-   state["m_numQuaternions"] = isd.at("NUMBER_OF_QUATERNIONS");
-
-   int numEphem = isd.at("NUMBER_OF_EPHEM");
-   for (int i=0;i < numEphem * 3; i++){
-       state["m_ephemPts"].push_back(isd.at("EPHEM_PTS").at(i));
-       state["m_ephemRates"].push_back(isd.at("EPHEM_RATES").at(i));
-   }
-
-   int numQuat = isd.at("NUMBER_OF_QUATERNIONS");
-   for (int i=0; i < numQuat * 4; i++){
-       state["m_quaternions"].push_back(isd.at("QUATERNIONS").at(i));
+   state["m_dtEphem"] = isd.at("dt_ephemeris");
+   state["m_t0Ephem"] = isd.at("t0_ephemeris");
+   state["m_dtQuat"] =  isd.at("dt_ephemeris");
+   state["m_t0Quat"] =  isd.at("t0_ephemeris");

```

Comment:
quaternion? - line above too.

Reply:
It depends on if we want to have position and pointing at different times. I could see requiring position and pointing to be at the same times.

If we go with possibly separate times then you're correct about using dt_quaternion and t0_quaternion